src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -99,7 +97,17 @@
 
 (rf/reg-event-fx :wallet/clean-selected-token
  (fn [{:keys [db]}]
-   {:db (assoc-in db [:wallet :ui :send :token] nil)}))
+   {:db (update-in db [:wallet :ui :send] dissoc :token :type)}))
+
+(rf/reg-event-fx :wallet/clean-selected-collectible
+ (fn [{:keys [db]}]
+   (let [type (get-in db [:wallet :ui :send :type])]
+     {:db (update-in db
+                     [:wallet :ui :send]
+                     dissoc
+                     :collectible
+                     :amount
+                     (when (= type :collecible) :type))})))
```

Comment:
Could we name it `send-type` (or something better?)  `type` is a clojure function, and we've had issues because we wrongly refer to a core fn instead of a var

Reply:
Yep! Makes sense, thanks for your suggestion. I'll change the key name